Add 2 news: 400 tutorial milestone & dada2 tutorial (#5008)
galaxyproject/training-material: news/_posts/2024-06-06-400-tutorials-milestone.md

---
title: "Celebrating the Milestone: 400 Tutorials on the Galaxy Training Network!"
layout: news
contributions:
  authorship:
  - bebatut
  - hexylena
  - shiltemann
tags:
  - gtn
abbreviations:
  GTN: Galaxy Training Network
cover: news/images/2024-06-06-400-tutorials.png
coveralt: 400 tutorials
---

The Galaxy Training Network (GTN) has reached an exciting milestone: **our 400th tutorial**! This achievement is a testament to the dedication and hard work of our community of educators, researchers, and developers over the last 9 years. 

The GTN was established to provide comprehensive and accessible training materials for users of the [Galaxy](https://galaxyproject.org/), a widely-used, open-source platform that empowers researchers worldwide to conduct data analysis. Over the years, the network has massively grown, both in content and community engagement, reflecting the dynamic nature of scientific research and the continuous need for up-to-date training resources.

> ![2 graphs representing the increase over times of (left) Materials (Tutorials + Slides) and (right) Contributors to the Galaxy Training Network]({{site.baseurl}}/news/images/2024-06-06-contributions.png)

Reviewer: ```suggestion
![2 graphs representing the increase over times of (left) Materials (Tutorials + Slides) and (right) Contributors to the Galaxy Training Network. each grow from ~50 in 2017 to ~500 and ~350 respectively, today]({{site.baseurl}}/news/images/2024-06-06-contributions.png)
```

---
title: "Celebrating the Milestone: 400 Tutorials on the Galaxy Training Network!"
layout: news
contributions:
  authorship:
  - bebatut
  - hexylena
  - shiltemann
tags:
  - gtn
abbreviations:
  GTN: Galaxy Training Network
cover: news/images/2024-06-06-400-tutorials.png
coveralt: 400 tutorials
---

> The Galaxy Training Network (GTN) has reached an exciting milestone: the **addition of our 400th tutorial**! This achievement is a testament to the dedication and hard work of our community of educators, researchers, and developers. 

Reviewer: ```suggestion
The Galaxy Training Network (GTN) has reached an exciting milestone: the **our 400th tutorial**! This achievement is a testament to the dedication and hard work of our community of educators, researchers, and developers over the last 9 years. 
```

The GTN was established to provide comprehensive and accessible training materials for users of the [Galaxy](https://galaxyproject.org/), a widely-used, open-source platform that empowers researchers worldwide to conduct data analysis. Over the years, the network has massively grown, both in content and community engagement, reflecting the dynamic nature of scientific research and the continuous need for up-to-date training resources.

![2 graphs representing the increase over times of (left) Materials (Tutorials + Slides) and (right) Contributors to the Galaxy Training Network]({{site.baseurl}}/news/images/2024-06-06-contributions.png)

## A Journey of Knowledge and Collaboration

From its inception, the GTN has been driven by the goal of making high-quality, user-friendly tutorials available to everyone. The diverse range of tutorials covers topics from basic Galaxy usage to advanced bioinformatics analyses, catering to both beginners and seasoned researchers.

> This 400th tutorial represents more than just a number; it embodies the collaborative spirit and the shared pursuit of knowledge that defines the Galaxy community. Each tutorial is meticulously crafted by experts who not only have a deep understanding of their subject matter but also a passion for teaching and community building.

Reviewer: ```suggestion
This 400th tutorial represents more than just a number; it embodies the collaborative spirit and the shared pursuit of knowledge that defines the Galaxy community. Each tutorial was contributed by experts who not only have a deep understanding of their subject matter but also a passion for teaching and community building.
```

## Highlights of Our Tutorials

> Our tutorials span a wide array of scientific domains and technical skills (30+ topics), including:

Reviewer: ```suggestion
Our tutorials span a wide array of scientific domains and technical skills (30+ topics), including but not limited to:
```

- **Genomics**: Explore the fundamentals of sequence analysis, variant calling, and genome assembly.
- **Proteomics**: Dive into protein identification, quantification, and functional analysis.
- **Ecology**: Learn to analyse Ecological data through Galaxy.
- **Machine Learning**: Understand the integration of machine learning techniques in biological data analysis.
- **Data Visualization**: Master the art of presenting your data in clear and informative ways.

Each tutorial is designed to be interactive, with hands-on exercises that allow users to apply what they've learned in real-world scenarios. Moreover, the tutorials are continuously updated to incorporate the latest advancements and tools in the field.

## The Power of Community

The GTN's success is a direct result of the vibrant and collaborative Galaxy community. Contributors from around the globe share their expertise, review content, and provide feedback to ensure the tutorials remain accurate and relevant. This collective effort has created a rich repository of knowledge that is freely accessible to anyone, anywhere. **We thank everyone who contributed!**

## Looking Ahead

> As we celebrate this milestone, we remain committed to expanding and enhancing our training materials. The journey doesn't end here—there are many more tutorials to create, more innovations to integrate, and more minds to inspire. We invite you to explore our tutorials, join our community, and contribute to the ever-growing Galaxy Training Network.

Reviewer: ```suggestion
As we celebrate this milestone, we remain committed to expanding and enhancing our training materials. We have recently expanded beyond Galaxy into Data Science and command line analyses, and we look forward to expanding to even more new life science topics going forward.
```

Here's to the next 400 tutorials and beyond—together, we are advancing science and education, one tutorial at a time!

---

> Whether you're a seasoned researcher or a newcomer to the field, the Galaxy Training Network is here to support your learning journey. Visit our [website]({% link index.md %}) to explore our tutorials and become part of our global community.

Reviewer: I'm not sure we need a call to action here, it is more of a thank you post, it could be fine to just end on "here's to the next"

Author: Let's remove it then
